docs/machine-learning/basic-concepts-model-training-in-mldotnet.MD

```diff
@@ -0,0 +1,169 @@
+---
+title: Basic concepts for model training in ML.NET
+description: This article explains the basic concepts covering *model training* scenarios in ML.NET. Not all concepts are relevant to the more simple scenario of *prediction with existing model*.
+ms.date: 11/06/2018
+ms.topic: overview
+ms.custom: mvc
+#Customer intent: As a developer, I want to learn the basic ML.NET concepts related to model training so that I can understand how to leverage machine learning in my applications.
+---
+# Basic concepts for model training in ML.NET
+
+This article explains the basic concepts covering *model training* scenarios in ML.NET. Not all concepts are relevant to the more simple scenario of *prediction with existing model*.
+
+## Model training concepts
+
+This document covers the following concepts related to model training in ML.NET:
+
+- [*Data*](#data), represented as an `IDataView` interface.
+  - In ML.NET, data is very similar to a SQL view: it's a lazily-evaluated, immutable, cursorable, heterogenous, schematized dataset. 
+  - An excellent document about the data interface is [IDataView Design Principles](https://github.com/dotnet/machinelearning/blob/master/docs/code/IDataViewDesignPrinciples.md).
+- [*Transformer*](#transformer), represented as `ITransformer` interface.
+  - In one sentence, a transformer is a component that takes data, does some work on it, and returns new 'transformed' data.
+  - For example, you can think of a machine learning model as a transformer that takes features and returns predictions.
+  - Another example, 'text tokenizer' would take a single text column and output a vector column with individual 'words' extracted out of the texts.
```

Comment:
This seems like an obscure example. Maybe HTML encoding of a string?

Reply:
No. This appears to be text analytics.